### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by RimuruDev** [vavilichev/Lukomor#8] [Q&A] [answered]
### Good day. I wanted to know if this project is alive, and if it is alive, will documentation be written for it. Thank you in advance for your response.

**Comment by vavilichev** [comment, marked as answer]
> ### Good day. I wanted to know if this project is alive, and if it is alive, will documentation be written for it. Thank you in advance for your response.

Hello! I am writing a big update for this project now. After that I am going to write a documentation.

**Comment by vovko21** [comment]
When the documentation will be added? At least scheme. It is not necessary to describe each class. I just want to get the point across.
